vehicles/car01.lua

```diff
@@ -0,0 +1,46 @@
+local definition = ...
+local types = {
+	"black", "blue", "brown", "cyan",
+	"dark_green", "dark_grey", "green",
+	"grey", "magenta", "orange",
+	"pink", "red", "violet",
+	"white", "yellow", "hot_rod",
+	"nyan_ride", "oerkki_bliss", "road_master"
+}
+
+for _, color in ipairs(types) do
+    if minetest.settings:get_bool("vehicle_mash.enable_" .. color .. "_car") then
+        local def = table.copy(definition)
+        local name = "car_"..color
+        local capitalization = color:gsub("^%l", string.upper)
+        local style = capitalization:gsub("_", " ")
+
+        def.description = vehicle_mash.S("@1 Car", style)
+
+        -- These types of car do not have an inventory image.
+        if color == "road_master" then
+            def.inventory_image = "inv_car_brown.png"
+            def.wield_image = "inv_car_brown.png"
```

Comment:
Idea: you could make use of a definition table above. It will be a bit verbose but very easy to scale without special code handling.
```Lua
local types = {
	black = { desc = S("Black car") }, -- defualt color "black"
	...
	road_master = { desc = S("Road master car"), color = "brown" } -- override color: "brown"
	...
}

...

for name, car in ipairs(types) do
	...
	def.description = car.desc;
	def.inventory_image = ("inv_car_%s.png"):format(car.color or name)
	...
```